events/views.py

```diff
@@ -25,21 +25,26 @@ def index(request):
     oldEvents = Event.objects.filter(
         active=True, eventDate__lt=now).order_by('-eventDate')
 
+    categories = Category.objects.all()
+
+    formFilter = FilterEventSearch()
```

Comment:
@mapeveri while there is already a lot of code which is not compliant with PEP8, I'd love to see all new code comply with PEP8 style guide. Like in using `form_filter` instead of `formFilter`.